charter.html

```diff
@@ -281,13 +295,15 @@ <h2 id=work-items>Work Items</h2>
 <p>The Chairs may add Work Items, but must not add Work Items which lack
 the support of at least two <a href=#implementer>implementers</a>.
 
-<p class=note>Typically, Work Items begin life as a Proposal before
-being adopted.
+<p>Artifacts of Work Items may include an explainer describing the
+proposed changes to the web platform as well as a Draft Community Group
+Report defining the proposed changes in detail.
 
 <p>When a Work Item's Editors deem the Work Item ready for migration to
 a standards body, they will notify the Chairs. The Editors and Chairs
-will then work together to identify the best destination, and will work
-with the destination standards body to successfully migrate the work.
+will then work together to produce a Final Community Group Report, to
+identify the best destination body, and will work with the destination
```

Comment:
This isn't quite parallel:

```suggestion
will then work together to produce a Final Community Group Report,
identify the best destination body, and work with the destination
```